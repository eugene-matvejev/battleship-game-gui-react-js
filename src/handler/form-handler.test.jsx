import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import FormHandler from './form-handler';

configure({ adapter: new Adapter() });

describe('<FormHandler/>', () => {
    const props = {
        config: [
            {
                items: [
                    {
                        c: (props) => <span {...props}/>,
                    }
                ]
            }
        ]
    };

    describe('render', () => {
        it('with default/required props', () => {
            const c = shallow(<FormHandler {...props} />);

            expect(c).toMatchSnapshot();
        });
    });
});
