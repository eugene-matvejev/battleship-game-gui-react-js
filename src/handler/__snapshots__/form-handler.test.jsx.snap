// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormHandler/> render with default/required props 1`] = `
<form
  className="form "
  data-cy=""
>
  <Accordion
    className=""
    data-cy="section-0"
    data-section={0}
    key="0"
    onCollapse={[Function]}
    qaSelector=""
  >
    <c
      data-field={0}
      data-section={0}
      key="0"
      qaSelector="section-0-input-0"
    />
  </Accordion>
</form>
`;
